### ISSUE TITLE
Training with smaller GPU memory?

### ISSUE THREAD

**Issue by dringlourious** [facebookresearch/DepthContrast#5]
Hi 
when the first time I train the model (using waymo datasets) with the default setting it works fine. But it was using GPU having 24G VRAM.

I would like to ask if I train it with a smaller GPU, eg. having 8G VRAM, how to change the config so that it is wont run out of VRAM? 

Thanks

**Comment by zaiweizhang** [comment]
You can decrease the batch size. That will decrease the usage of VRAM. Please change the learning rate accordingly. (If you decrease the batch size by 2, then decrease the learning by 2.)
